src/components/NameForm.tsx

```diff
@@ -1,9 +1,47 @@
 import React, {ChangeEventHandler, EventHandler, FormEventHandler, MouseEventHandler, SyntheticEvent} from "react";
 import ReactDOM from "react-dom";
+import styled, {keyframes} from "styled-components";
 
 type Props = { callback: (arg: number) => void };
 type State = { value: number };
 
+
+const Button = styled.button`
```

Comment:
Those components should be in separate files for now.
please create /styled folder in components and move them there.

Reply:
reference to architecture:
https://hs3city.slack.com/archives/C018ZBP0FK5/p1604669788022000

Reply:
Got it